File: app/src/main/java/com/nuxflix/app/Movie.java
package com.nuxflix.app;

public class Movie {
    String title;
    String description;
    String year;
    String duration;
    String rating;
    String videoUrl;
    String posterUrl;
    int posterColor;

    public Movie(String title, String description, String year, String duration, String rating, String videoUrl, String posterUrl, int posterColor) {
        this.title = title;
        this.description = description;
        this.year = year;
        this.duration = duration;
        this.rating = rating;
        this.videoUrl = videoUrl;
        this.posterUrl = posterUrl;
        this.posterColor = posterColor;
    }
}

File: app/src/main/java/com/nuxflix/app/Movie.kt
package com.nuxflix.app

data class Movie(
    val id: Int,
    val title: String,
    val description: String,
    val year: String,
    val duration: String,
    val rating: String,
    val videoUrl: String,
    val videoUrlBackup: String = "",
    val posterUrl: String,
    val posterColor: Int = 0xFF1A1A2E.toInt(),
    val category: String = "أفلام جديدة",
    val isFeatured: Boolean = false,
    val trailerUrl: String = ""
)